docs/formulating_requests.md

```diff
@@ -0,0 +1,61 @@
+To be able to make a request, you need to decide on a few main points:
+
+- Which variables do you want?
+    - And in which dataset are these variables located? (E.g. ERA5-Land).
+- For which years do you want the variable?
+- Do you want to the data globally, or specify a bounding box (using `--area`).
+- (*For pressure level data:*) on which levels do you want the data.
+
+???+ note
+    All available arguments are listed in the [CLI Reference](reference/arguments.md)
+
+
+
+### Example request
+
+A typical request in `era5cli` can look like the following:
+
+```
+era5cli hourly --land --variables 2m_temperature 2m_dewpoint_temperature
+    --startyear 2000 --endyear 2020 --splitmonths True
+    --area 53.6 3.3 50.7 7.5
```

Comment:
Consider one option per line, might be easier for a new user to see what's going on

```suggestion
era5cli hourly \
    --land \
    --variables 2m_temperature 2m_dewpoint_temperature \
    --startyear 2000 \
    --endyear 2020 \
    --splitmonths True \
    --area 53.6 3.3 50.7 7.5
```

Reply:
Nice, thanks for the suggestion. It's helpful that with the backslash the command is still copy-pasteable